docs/Tutorials/Examples/Privacy-Example.md

```diff
@@ -97,87 +83,82 @@ For more information on the endpoints and services, refer to README.md in the in
 ****************************************************************
 ```
 
-## Send private transactions and read values
+## Deploy the private contract and interact with the nodes
 
-The Event Emitter script deploys a contract with a privacy group of Node1 and Node2. That is, the
-other nodes cannot access the contract. After deploying the contract, Event Emitter stores a value.
+To deploy a private contract to another [privacy group](../../Concepts/Privacy/Privacy-Groups.md) member, use the
+[web3js-eea](https://github.com/ConsenSys/web3js-eea) library and
+the [`eea_sendRawTransaction`](../../Reference/API-Methods.md#eea_sendrawtransaction) API call.
+You must use this API call instead of [`eth_sendTransaction`](https://eth.wiki/json-rpc/API) because Hyperledger Besu
+keeps account management separate for stronger security.
 
-In the `web3js-eea` directory, run `eventEmitter.js`:
+This example uses the [web3js](https://www.npmjs.com/package/web3) library to make the API calls, the example
+creates three Besu nodes, with each node having a corresponding Tessera node for privacy. You can access the Besu
+member nodes for API calls on the following ports:
 
 ```bash
-node example/eventEmitter.js
-```
+Member1Besu RPC: http://localhost:20000
+Member1Tessera: http://localhost:9081
 
-!!! tip
+Member2Besu RPC: http://localhost:20002
+Member2Tessera: http://localhost:9082
```

Comment:
@joshuafernandes I updated the member names for the Tessera nodes. Let me know if this was the correct thing to do.

Reply:
Yep, good catch :) Nice spotting @bgravenorst !